blog/tests/test_search.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from search.models import Search
+
+from ..models import Content
+
+pytestmark = pytest.mark.django_db
```

Comment:
Humm, I don't think so because it's on the first level of the module, so it can be imported from elsewhere.